02week/pigLatin.js

```diff
@@ -7,24 +7,51 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
-
-function pigLatin(str) {
-  function check(obj) {
-  	return ['a','i','u','e','o'].indexOf(str.charAt(obj)) == -1 ?
-      										  check(obj+1) : obj;}
+//old code
```

Comment:
@JacobGonz1993 
Please remove old code or big comment blocks of unused code before you make a PR, thanks!